modules/widgets/simple-payments.php

```diff
@@ -0,0 +1,330 @@
+<?php
+/*
+Plugin Name: Simple Payments
+Description: Simple Payments button implemented as a widget.
+Version: 1.0
+Author: Automattic Inc.
+Author URI: http://automattic.com/
+License: GPLv2 or later
+*/
+
+function jetpack_register_widget_simple_payments() {
+	register_widget( 'Simple_Payments_Widget' );
+}
+add_action( 'widgets_init', 'jetpack_register_widget_simple_payments' );
+
+class Simple_Payments_Widget extends WP_Widget {
+	private static $dir              = null;
+	private static $url              = null;
+	private static $product_defaults = null;
+
+	function __construct() {
+		$widget = array(
+			'classname'   => 'simple-payments',
+			'description' => __( 'Add a simple payment button.', 'jetpack' ),
+		);
+
+		parent::__construct(
+			'Simple_Payments_Widget',
+			apply_filters( 'jetpack_widget_name', __( 'Simple Payments', 'jetpack' ) ),
```

Comment:
Could you add a docblock here, to make our parser happy? This should work:
`/** This filter is documented in modules/widgets/facebook-likebox.php */`